src/pages/login/login.js

```diff
@@ -45,7 +49,7 @@ export default function Login() {
             </div>
                             <a href="#">Forgot your password?</a>
                         </div>
-                        <button onClick={e => handleLogin(e)} className="btn login">Login</button>
+                        <button onClick={e => handleSubmit(e)} className="btn login">Login</button>
```

Comment:
{handleSubmit} is also the same thing rather than a callback when the parameter is only the event

Reply:
Done, thanks for suggesting.